WebRoot/templates/prefs/Pages.template

```diff
@@ -2207,22 +2207,24 @@ NOTE: The tables inside of the filter rule sections MUST contain a tbody element
 <template id='prefs.Pages#GrantRightsDialog'>
 	<table role="presentation">
 		<tr id='${id}_name_row'>
-			<td class="Label" style="padding-right:10px"><$=ZmMsg.emailLabel$></td>
+			<td class="Label"><$=ZmMsg.emailLabel$></td>
+		</tr>
+		<tr>
 			<td id="${id}_name_cell">
 			</td>
 		</tr>
 		<tr>
-			<td class="Label">&nbsp;</td>
+			<td class="Label"></td>
 			<td>
 				<table role="presentation">
```

Comment:
I don't feel need to have two separate classes `SendAs` and `SendObo` to `TD`. I guess you have added these class to target an TD, and same styling is needed for TD in this case. In such cases, you could have targeted TD from TABLE as well, so when new row gets added, we don't have to figure out and add class for TD's of new TR.

Also you can have kept same className for TD's as well.

Thanks